dashboard/app/models/script.rb

```diff
@@ -1510,6 +1510,13 @@ def assignable_info
     end
     info[:is_stable] = true if is_stable
 
+    # For CSF scripts, set the assignment family category so the CSF courses will be
+    # grouped together under the 'CS Fundamentals' heading in the course assignment dropdown.
+    if ScriptConstants.csf_script?(self)
```

Comment:
this approach seems fine for now. in the future I imagine that we'd change this to look at some "category" property on the Course Offering.